avaframe/in2Trans/geoTrans.py

```diff
@@ -149,8 +233,8 @@ def checkProfile(AvaProfile, projSplitPoint):
 
 def bresenham(x0, y0, x1, y1, cs):
```

Comment:
Maybe rename to something along the line findCellsCrossedByLine_Breseham?  So if someone sees this call in another function it is easy to get an idea of what is happening?